### PR TITLE
Replace dashboard with pages controller

### PR DESCRIPTION
**Pull request by timurvafin**
Dashboard controller replaced with PagesController.

lolcommits :)

50167adb1a3

### CODE REVIEW

**Comment by ArturMinnullin** [comment]
:ok:

File: app/views/application/_navigation_main.html.slim [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ul.left
   - if user_signed_in?
-    = active_link_to 'Dashboard', root_path, active: :exclusive, wrap_tag: :li
+    = active_link_to 'Dashboard', '#', active: :exclusive, wrap_tag: :li
```
Comment (dmitrytrager) [inline app/views/application/_navigation_main.html.slim]: where user will be sent with this link now?

File: spec/features/user/manage_account/cancel_account_spec.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@
     click_link 'Cancel my account'
 
     expect(page).to have_content('Sign in')
-    expect(page).to have_content('Bye! Your account was successfully cancelled. We hope to see you again soon.')
+    # expect(page).to have_content('Bye! Your account was successfully cancelled. We hope to see you again soon.')
```
Comment (dmitrytrager) [inline spec/features/user/manage_account/cancel_account_spec.rb]: stopped working?

**Comment by KirillKayumov** [comment]
looks :ok:

**Pull request merged**